core/src/main/java/greencity/repository/HabitDictionaryTranslationRepo.java

```diff
@@ -38,9 +38,11 @@
      * @param language code language.
      * @return List of available {@link HabitDictionaryTranslation}`s.
      */
-    @Query(nativeQuery = true, value = "SELECT *  FROM habit_dictionary_translation WHERE"
-        + " habit_dictionary_id NOT IN\n"
-        + " (SELECT habits.habit_dictionary_id FROM habits WHERE user_id = ?1 AND status = 'true')\n"
+    @Query(nativeQuery = true, value = "SELECT *  FROM habit_dictionary_translation WHERE "
+        + "habit_dictionary_id NOT IN "
```

Comment:
![image](https://user-images.githubusercontent.com/3871580/91343117-b563d580-e7e4-11ea-95c6-2009b4c75aa8.png)
Is that what you are trying to achieve with this NOT IN?